### ISSUE TITLE
Happy path E2E tests failed on Eclipse Che installation step because of TLS certificate exception

### ISSUE THREAD

**Issue by dmytro-ndp** [eclipse-che/che#16292]
### Describe the bug
Happy path tests failed:
```
[21:35:11] ✅  Post installation checklist [failed]
[21:35:11] → ERR_TIMEOUT: Timeout set to pod ready timeout 180000
 ›   Error: Error: ERR_TIMEOUT: Timeout set to pod ready timeout 180000
 ›   Installation failed, check logs in '/tmp/chectl-logs/1583803781444'
```

#### [Eclipse Che server log](https://codeready-workspaces-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/che-pr-tests/view/K8S/job/Multiuser-Che-PR-check-E2E-Happy-path-tests-against-k8s/2336/artifact/logs-and-configs/che-logs/che-5dd575994f-gmdv8.pod.log) 
```
2020-03-10 06:02:24,244[ost-startStop-1]  [INFO ] [o.e.c.m.k.s.KeycloakSettings 91]     - Retrieving OpenId configuration from endpoint: https://keycloak-che.10.0.101.177.nip.io/auth/realms/che/.well-known/openid-configuration
2020-03-10 06:02:24,482[ost-startStop-1]  [ERROR] [o.a.c.c.C.[.[localhost].[/api] 175]  - Exception sending context initialized event to listener instance of class [org.eclipse.che.inject.CheBootstrap]
com.google.inject.CreationException: Unable to create injector, see the following errors:

1) Error injecting constructor, java.lang.RuntimeException: Exception while retrieving OpenId configuration from endpoint: https://keycloak-che.10.0.101.177.nip.io/auth/realms/che/.well-known/openid-configuration
  at org.eclipse.che.multiuser.keycloak.server.KeycloakSettings.<init>(KeycloakSettings.java:70)
  at org.eclipse.che.multiuser.keycloak.server.KeycloakSettings.class(KeycloakSettings.java:53)
  while locating org.eclipse.che.multiuser.keycloak.server.KeycloakSettings
    for the 1st parameter of org.eclipse.che.multiuser.keycloak.server.KeycloakProfileRetriever.<init>(KeycloakProfileRetriever.java:40)
  at org.eclipse.che.multiuser.keycloak.server.KeycloakProfileRetriever.class(KeycloakProfileRetriever.java:33)
  while locating org.eclipse.che.multiuser.keycloak.server.KeycloakProfileRetriever
...
Caused by: javax.net.ssl.SSLHandshakeException: java.security.cert.CertificateException: No subject alternative DNS name matching keycloak-che.10.0.101.177.nip.io found.
	at sun.security.ssl.Alerts.getSSLException(Alerts.java:192)
...
Caused by: java.security.cert.CertificateException: No subject alternative DNS name matching keycloak-che.10.0.101.177.nip.io found.
	at sun.security.util.HostnameChecker.matchDNS(HostnameChecker.java:214)
```

### Possible reason
Incorrect custom-resource.yaml with `tlsSupport: true` and `selfSignedCert: false`.

### Che version

 - [ ] latest
 - [x] nightly
 - [ ] other: please specify

### Steps to reproduce
```
chectl server:start --k8spodreadytimeout=180000 --installer=operator --listr-renderer=verbose --platform=minikube --che-operator-cr-yaml=custom-resource.yaml
```
Where custom-resource.yaml has been given from https://raw.githubusercontent.com/eclipse/che-operator/master/deploy/crds/org_v1_che_cr.yaml

### Runtime
   - [ ] kubernetes (include output of `kubectl version`)
   - [ ] Openshift (include output of `oc version`)
   - [x] minikube 1.1.1
   - [ ] minishift (include output of `minishift version` and `oc version`)
   - [ ] docker-desktop + K8S (include output of `docker version` and `kubectl version`)
   - [ ] other: (please specify)

### Installation method
   - [x] chectl/0.0.20200306-next.7b420cf linux-x64 node-v10.19.0
   - [ ] che-operator 
   - [ ] minishift-addon
   - [ ] I don't know

### Environment
   - [ ] my computer
       - [ ] Windows
       - [ ] Linux
       - [ ] macOS
   - [ ] Cloud
       - [ ] Amazon
       - [ ] Azure
       - [ ] GCE
       - [ ] other (please specify)
   - [x] other: CRW CCI

**Comment by dmytro-ndp** [comment]
Well, there is another issue which blocks execution of Happy path tests test pipelines against Eclipse Che with TLS support on minikube: [Eclipse Che pod bootstrap timeout on chectl install, when using Che operator with TLS and unsigned certificate on non-OpenShift kube](https://github.com/eclipse/che/issues/16280).

**Comment by dmytro-ndp** [comment]
It was decided to run Happy path tests against Eclipse Che without TLS support to unblock PR check until https://github.com/eclipse/che/issues/16280 will be fixed.
